indico/modules/events/management/controllers/protection.py

```diff
@@ -147,3 +148,10 @@ class RHCategoryRolesJSON(RHManageEventBase):
     def _process(self):
         category_roles = CategoryRole.get_category_roles(self.event.category)
         return jsonify([serialize_category_role(cr, legacy=False) for cr in category_roles])
+
+
+class RHRegistrationFormsJSON(RHManageEventBase):  # TODO: Give this the API prefix instead of JSON?
```

Comment:
I wonder if this should stay here or go in the registrations module...

Reply:
I had it like that before, until I noticed both categories and event roles were here.
I also don't like to see it here, I'll get them back. Not sure if we should do something about the others tho.

Reply:
hm, category roles don't really have a better place... maybe keeping it here for now is fine since it's somewhat generic and used everywhere in events, not just in registration forms. that way all of these endpoints are next to each other